net/https-dns-proxy/files/https_dns_proxy.config

```diff
@@ -1,8 +1,19 @@
 config https_dns_proxy
 	option listen_addr '127.0.0.1'
 	option listen_port '5053'
+	option bind_addr '8.8.8.8,8.8.4.4'
+	option url_prefix 'https://dns.google.com/resolve?'
+	option user 'nobody'
+	option group 'nogroup'
+	option subnet_addr ''
+	option proxy_server ''
+
+config https_dns_proxy
```

Comment:
I'd rather not run two of these. I imagine it's pretty unlikely that users will want both at the same time so it's just wasting resources running both.

Reply:
I, for one, subscribe to "two is one, one is none" principle, so I do have two instances running and dnsmasq pointed to both instances. That was my biggest gripe with the `dnscrypt-proxy` back in the day when it would only allow one instance, because even the more reliable ones sometimes would go down.
I would rather have both providers in the config to make for a more reliable and fault-tolerant setup. That's my last plea to you to reconsider, if you're still very strongly opposed to a more fault-tolerant setup for a small resource gain, let me know, I'll comment Cloudflare out in the default config.

Reply:
I don't feel very strongly about this and you clearly do. You also make a good point about redundancy and this plays also into not playing any favoritism with default services so go ahead.